config/optional/block.block.umami_disclaimer.yml

```diff
@@ -16,4 +16,10 @@ settings:
   label: 'Umami disclaimer'
   provider: demo_umami_content
   label_display: '0'
+  umami_disclaimer:
+    value: '<strong>Umami Magazine & Umami Publications</strong> is a fictional magazine and publisher for illustrative purposes only.'
+    format: 'basic_html'
+  umami_copyright:
+    value: '© 2017 Terms & Conditions'
```

Comment:
Should this be _&copy;_ instead of the copyright symbol? I'm not sure if that would cause it to display as &copy; though.